src/validator.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+const validator = (text, caseToTransform) => {
+  const errors = [];
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Move `availableCases` outside the function and the constant should be in the upper case